xaitk_saliency_demo/app/xaitk_widgets.py

```diff
@@ -146,6 +153,42 @@ def __init__(self):
                 )
 
 
+class AddModelDialog(vuetify.VDialog):
+    def __init__(self, add_model):
+        super().__init__(v_model=("show_add_model", False), width="500")
+        with self:
+            with vuetify.VCard():
+                with vuetify.VCardTitle() as header:
+                    header.add_child("Add Hugging Face Hub Model")
+                with vuetify.VCardText():
+                    with html.P(
+                        "Enter the name of an image model hosted on the",
+                        style="padding-bottom: 10px;",
+                    ):
+                        html.A(
+                            "Hugging Face Hub",
+                            href="https://huggingface.co/models?pipeline_tag=image-classification&sort=trending",
+                            target="_blank",
+                        )
+                    vuetify.VTextField(
+                        label="Model Name", v_model=("new_model_name", "")
+                    )
+                with vuetify.VCardActions():
+                    vuetify.VSpacer()
+                    with vuetify.VBtn(
```

Comment:
swap order of the 2 button? `[Ok] [Cancel]`?

Reply:
done